.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+  - 4
+  - 6
```

Comment:
Since Node 8 is out and LTS coming up, you probably want to either add `stable` or `8` to test against that as well.